modules/serverless-rn-known-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/release-notes.adoc
+
+[id="serverless-rn-known-issues_{context}"]
+= Known issues
+
+////
+Consequence - What user action or situation would make this problem appear? What did the customer experience as a result of the issue? What was the symptom?
+Cause (if it has been identified) - Why did this happen?
+Workaround (If there is one)- What can you do to avoid or negate the effects of this issue in the meantime?  Sometimes if there is no workaround it is worthwhile telling readers to contact support for advice. Never promise future fixes.
+Result - If the workaround does not completely address the problem.
+////
+
+These limitations exist in {ProductName} at this time:
+
+* The Knative Serving Operator should wait for ServiceMeshMemberRoll to include the `knative-serving` namespace. Currently, we recommend creating the `knative-serving` namespace and then installing the operator. Istio does not consider the `knative-serving` namespace to be in the ServiceMeshMemberRoll when the Knative Serving pods are being created. Consequently, the sidecars are not injected.
+
+* Knative service returns a 503 status code when the cluster is running for a long time. The Knative Serving pods do not show any errors. Restarting the `istio-pilot` pod temporarily fixes the issue. 
```

Comment:
s/pods/Pods/

Reply:
We've been capitalizing _Pod_ per our guidelines: https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/term_glossary.adoc#pod